### PR TITLE
Added shoppingCart and cartItem entities with services and shoppingCa…

### PR DESCRIPTION
**Pull request by dima-semeniuk**
…rt controller

### CODE REVIEW

File: src/main/java/mate/academy/webapp/model/User.java [hunk 1]
```diff
@@ -43,6 +45,8 @@ public class User implements UserDetails {
 
     private String shippingAddress;
 
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
     @ManyToMany(fetch = FetchType.EAGER)
```
Comment (d1sam) [inline src/main/java/mate/academy/webapp/model/User.java:50]:
```suggestion
    @ToString.Exclude
    @EqualsAndHashCode.Exclude
    @ManyToMany(fetch = FetchType.LAZY)
```
also not related to this pr.

File: src/main/java/mate/academy/webapp/repository/ShoppingCartRepository.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package mate.academy.webapp.repository;
+
+import java.util.Optional;
+import mate.academy.webapp.model.ShoppingCart;
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
```
Comment (d1sam) [inline src/main/java/mate/academy/webapp/repository/ShoppingCartRepository.java:8]:
```suggestion
@Repository
public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
```

File: src/main/resources/db/changelog/changes/11-creaate-cartitems-table.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (d1sam) [inline src/main/resources/db/changelog/changes/11-creaate-cartitems-table.yaml:15]:
```suggestion
```

File: src/main/resources/db/changelog/changes/10-create-shopping_carts-table.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (d1sam) [inline src/main/resources/db/changelog/changes/10-create-shopping_carts-table.yaml:15]:
```suggestion
```

**Review by d1sam — CHANGES_REQUESTED** [review]
good job, but some changes should be done!

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**